build.gradle.kts

```diff
@@ -1,5 +1,5 @@
 
-version = "0.1.1"
+version = "0.1.7"

```

Comment:
I left out the version number bump and the jitpack stuff, just because my own version would still be 0.1.2 at whatever next version I end up bumping to, and because I already have a CI build, but for GitHub Actions/Packages instead.